### ISSUE TITLE
Vision models

### ISSUE THREAD

**Issue by maw-out** [SeargeDP/ComfyUI_Searge_LLM#13]
Thank you for your time, your nodes are great because when they are not used in the workflow, they free up RAM.

It would be great to also have the ability to read images with vision models (like Llava, for example).

**Comment by Santhanam1** [comment]
yeah! llama-cpp-python does support multimodal, it would be cool to use it for LTX frame interpolation, hope it possible to add this feature
